src/Fixer/Phpdoc/PhpdocLineSpanFixer.php

```diff
@@ -105,6 +105,14 @@ public function isCandidate(Tokens $tokens): bool
     protected function createConfigurationDefinition(): FixerConfigurationResolverInterface
     {
         return new FixerConfigurationResolver([
+            (new FixerOptionBuilder('class', 'Whether class/interface/trait blocks should be single or multi line.'))
+                ->setAllowedValues(['single', 'multi', null])
+                ->setDefault('multi')
+                ->getOption(),
+            (new FixerOptionBuilder('use_trait', 'Whether trait usage blocks should be single or multi line.'))
+                ->setAllowedValues(['single', 'multi', null])
+                ->setDefault('single')
```

Comment:
Is it ok to use `single` as default here? I think most people use single line phpdocs for trait usages.

```php
class Foo
{
    /** @use Bar<string, int> */
    use Bar;
}
```

Or should all new config options default to `null` (so no fixes)?

Reply:
Changed defaults to `single` for `trait_import` and `other`.